pom.xml

```diff
@@ -500,6 +507,13 @@
                     <configuration>
                         <source>${project.build.targetJdk}</source>
                         <target>${project.build.targetJdk}</target>
+                        <annotationProcessorPaths>
+                            <annotationProcessorPath>
+                                <groupId>org.immutables</groupId>
+                                <artifactId>value</artifactId>
+                                <version>${dep.immutables.version}</version>
+                            </annotationProcessorPath>
+                        </annotationProcessorPaths>
```

Comment:
Is this annotation processor config still required?

Reply:
Yes, at least Eclipse does not seem to consider test-only dependencies for annotation processing, and this tells it to actually use it.  I think javac may not need it but it's a PITA to not have your IDE work.